app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="authentication-form">
+<h2>Sign up</h2>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+
+</div>
```

Comment:
- [ ] great job getting your authentication to work. As requested by the previous reviewer [here](https://github.com/Rafiduzzaman/blog-app/pull/9#pullrequestreview-1795216180), kindly ensure your login and signup pages elements are aligned as the given wireframe.

Your design
![image](https://github.com/Rafiduzzaman/blog-app/assets/81932478/21cbf551-c8ad-43cd-b57d-adac8a11864b)

given frame
![image](https://github.com/Rafiduzzaman/blog-app/assets/81932478/3a534174-9021-49dc-8711-bde5f41eb98b)